docs/index.bs

```diff
@@ -3176,6 +3190,29 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
               1. Let |deserializedError| be the result of [=deserialize a serialized abort reason=] given null and |workerRealm|.
               1. [=fetch controller/Abort=] |preloadFetchController| with |deserializedError|.
       1. Else, resolve |preloadResponse| with undefined.
+      1. Set |response| to the result of [=Create Fetch Event and Dispatch=] with |request|, |registration|, |useHighResPerformanceTimers|, |timingInfo|, |workerRealm|, |reservedClient|, and |preloadResponse|.
+      1. Retrun |response|.
```

Comment:
Nit: collapse this into a single step, "Return the result of Create Fetch Event and Dispatch with..."

Do the same with "Set _response_ to the result of dequeue queue / Return response".

Then delete "Let response be null" from the start of the algorithm, and change "Output" to just "a response".